README.md

```diff
@@ -38,8 +40,7 @@ Select 1:
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
-```
+Variables declared outside of a function are global scopes. Global scopes can be accessed from anywhere in code. Local scopes are created by functions and can be accessed from within.
 
```

Comment:
That's part of it...
Variables created without the var keyword, no matter where in a program, are placed in the global scope.
Variables created with the var keyword are created in the current local scope.
All functions (and only functions) create a new local scope.
The current scope includes all outer (enclosing) scopes.